scripts/enemy.gd

```diff
@@ -1,6 +1,6 @@
 extends RigidBody2D
 
-var target = Vector2(400,400)
+@export var target : Vector2 = Vector2(0,0)
 var speed = 1000
```

Comment:
Please export the `speed` and `acc` variables as well, and let's try to use clearer names like `acceleration` instead of `acc` for the future

Reply:
entirely valid